script.js

```diff
@@ -0,0 +1,101 @@
+const submit = document.getElementById('submit');
+
+const myBook = [];
+
+function Books(title, author) {
+  this.title = title;
+  this.author = author;
+}
+
+function addBook(newBook) {
+  myBook.push(newBook);
+}
+
+function displayBook(newBook) {
+  const list = document.querySelector('#book-list');
+  const row = document.createElement('div');
+  row.innerHTML = `
+    <h4>${newBook.title}</h4>
+    <h4>${newBook.author}</h4>
+    <button><a href="#" class="btn btn-danger btn-sm remove">Remove</a></button><hr>`;
+
+  list.appendChild(row);
+
+  const bookIndex = myBook
+    .map((book) => book.title)
+    .indexOf(newBook.title);
+    // update book read state in local storage
+  localStorage.setItem(bookIndex.toString(), JSON.stringify(newBook));
+
+  // remove button
+  const removeBtn = row.querySelector('#book-list');
+  // removes from view
+  removeBtn.addEventListener('click', () => {
+  // removes from array
+    const bookIndex = myBook
+      .map((book) => book.title)
+      .indexOf(newBook.title);
+    myBook.splice(bookIndex, 1);
+    // remove book from local storage
+    localStorage.removeItem(bookIndex);
+  });
+}
```

Comment:
Good job here 👍🏽 

- [ ] create a function ```removeBook(id)``` and pass it down to the event listener like this ```document.addEventListener('click', removeBook)```